_docs_integrate/create-attribute-for-yourself.md

```diff
@@ -13,13 +13,112 @@ properties:
   - component: integrate
   - level: Beginner
   - implementation status: DONE
-  - documentation status: OPEN
+  - documentation status: DONE
   - published:
-  - link: integrate/create-attribute-for-yourself
+  - link: create-attribute-for-yourself
 require:
   - integrate/identityattribute-introduction
 required_by:
   - integrate/test-your-requests-validity
+# End automatic generation
 ---
 
-{% include scenarios/integrate/create-attribute-for-yourself.md %}
```

Comment:
The content of the "Create Attribute for yourself" scenario has been inserted here. However, the corresponding scenario from _includes/scenarios/integrate has not been deleted. The same applies to the following scenarios and use cases:

- (Szenario: Create Attribute for yourself)
- Szenario: Establish a Relationship to another Identity
- Szenario: Succeeding Attributes to update their values
- Use case: Create a RepositoryAttribute
- Use case: Create and share a RelationshipAttribute
- Use case: Get shared versions of a RepositoryAttribute
- Use case: Get versions of an Attribute
- Use case: Notify peer about RepositoryAttribute succession
- Use case: Share a RepositoryAttribute
- Use case: Succeed a RelationshipAttribute and notify peer
- Use case: Succeed a RepositoryAttribute
- Use case: Revoke RelationshipChange

Why is this the case for these scenarios and use cases? I would assume that this has something to do with the fact that the page content of these scenarios and use cases has only recently been added.